pkg/webreg/webreg.go

```diff
@@ -498,6 +498,28 @@ either:
   </li>
 </ul>
 
+<h4 id="buildroot"><a href="#buildroot">Buildroot Image</a></h4>
+<p>
+The buildroot image must contain all dependencies for building executables and
```

Comment:
Can we move this section below the `"artifacts"` one? We introduce `build_root` there and it makes sense to into more depth there

Reply:
That's what I did initially, and I believe the current state is better. Build root is something between an input and a built artifact - it is special in a way that it can be both built and imported. Half of the section describes what the build root needs to contain, which is unrelated to building artifacts.

Having it separate also allows it to have an header with an anchor so it's linkable (custom buildroot is not that rare #forum topic) and not mixed with other sections. I think that the two sections together would be unnecessarily long.

So if you insist then yes, but I ask you to consider ^^^ ;)